### DISCUSSION TITLE
Why the plugin does not work after installation?

### DISCUSSION THREAD

**Discussion by aaangeelo** [vslinko/obsidian-outliner#118] [Q&A] [answered]
HI! I'm literally new to obsidian and I'm still trying things out. Saw this plug-in while browsing the community plug-ins and immediately installed it and tried it out. However it does not work like how it's supposed to work, like the lines aren't showing. Idk how to solve this, please help. thank you.
![image](https://user-images.githubusercontent.com/83688406/117099186-1fd10a80-ada3-11eb-8cf6-80bdc2c19e57.png)

this is how it looks in preview mode
![image](https://user-images.githubusercontent.com/83688406/117099214-32e3da80-ada3-11eb-9679-b2522618e9ca.png)

Thanks so much

**Comment by aaangeelo** [comment, marked as answer]
OKAY! SO I JUST SAW THE PLUG-IN SETTING, I'M SO DUMB HAHAHAHA... IT HAS ALREADY BEEN SOLVED THANK YOU! KEEPING THIS HERE IN CASE SOMEONE ALSO ENCOUNTERS THE SAME PROBLEM AS MINE! HAHAHAHA THANKS SO MUCH

**Comment by ml81f5v3n** [comment]
Hello, I've installed AND enabled the plugin but it's not working for me.  The screenshots in the original post above are what I'm seeing, even after enabling the plugin in settings.  What am I missing?  Thanks.

**Reply by vslinko** [reply]
Hello. Based on your description, everything should work fine. Could you add more details and create an issue? https://github.com/vslinko/obsidian-outliner/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D

**Comment by andreaaa9** [comment]
If anyone else is having this problem it doesn't work in the new mode if you're  using an insider build! So toggle to edit mode to see it.
